tests/aws-cpp-sdk-core-tests/smithy/client/SmithyClientTest.cpp

```diff
@@ -8,9 +8,17 @@
 #include <smithy/identity/auth/built-in/SigV4AuthScheme.h>
 
 
+#include <smithy/identity/auth/built-in/SigV4aAuthSchemeResolver.h>
+#include <smithy/identity/auth/built-in/SigV4aAuthScheme.h>
 #include <aws/core/client/ClientConfiguration.h>
 
 #include "aws/core/endpoint/EndpointProviderBase.h"
```

Comment:
looks like this was missed somewhere before, please clean up using `<>`

Reply:
ack